argus/action_manager/windows.py

```diff
@@ -120,32 +121,84 @@ def get_installation_script(self):
         self.download_resource("windows/installCBinit.ps1",
                                r"C:\installCBinit.ps1")
 
+    def _execute(self, cmd, count=util.RETRY_COUNT, delay=util.RETRY_DELAY,
+                 command_type=util.CMD):
+        """Execute until succeeds and return only the standard output."""
+        stdout, _, _ = self._client.run_command_with_retry(
+            cmd, count=count, delay=delay, command_type=command_type)
+        return stdout
+
+    def check_cbinit_installation(self):
+        """Check if Cloudbase-Init was installed successfully."""
+        LOG.info("Checking Cloudbase-Init installation.")
+
+        try:
+            python_dir = introspection.get_python_dir(self._execute)
+        except exceptions.ArgusError as exc:
+            LOG.warning("Could not check Cloudbase-Init installation: %s", exc)
+            return False
+
+        check_cmd = r'& "{}\python.exe" -c "import cloudbaseinit"'.format(
+            python_dir)
+        try:
+            self._client.run_remote_cmd(
+                cmd=check_cmd, command_type=util.POWERSHELL)
+        except exceptions.ArgusError as exc:
+            LOG.debug("Cloudbase-Init installation failed: %s", exc)
+            return False
+
+        LOG.info("Cloudbase-Init was successfully installed!")
+        return True
+
+    def cbinit_cleanup(self):
```

Comment:
Do you mean, `_deploy_using_scheduled_task` method could need some additional specific clean up?